### PR TITLE
Top-K-Accuracy Score

### PR DESCRIPTION
**Pull request by vijay-jaisankar**
ref: #15 

## Key features of this PR:
- Exposing the metric from `scikit-learn`
- Tests for the same

## Notes
- Relative imports were used during testing as this method has not been merged into the module yet
- The `__init__.py` spec has been updated to include this metric.

### CODE REVIEW

File: tests/test_classification_accs.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import unittest
 import numpy as np
+import sys
 
-from mindsdb_evaluator.accuracy.classification import evaluate_multilabel_accuracy
+sys.path.insert(0, "../mindsdb_evaluator/accuracy")
```
Comment (paxcema) [inline tests/test_classification_accs.py]: Replying to your comment in Slack:

> Due to the new metric not being included yet in the module, the tests had to be run with sys.path mangling - this method worked on my machine but doesn't work in the Github action.

The GitHub actions should work for this branch, because you _have_ included the metric in the module, so this bit is unnecessary. Doing:

> from mindsdb_evaluator.accuracy.classification import evaluate_multilabel_accuracy, evaluate_top_k_accuracy

Should be enough. Looking at the commit history it doesn't seem like you've tried that, could you give it a go?

**Review by paxcema — CHANGES_REQUESTED** [review]

File: tests/test_classification_accs.py (hunk 1 above)
Comment (vijay-jaisankar) [reply tests/test_classification_accs.py]: Hello @paxcema, thanks for the feedback! https://github.com/mindsdb/mindsdb_evaluator/pull/16/commits/388ea60ab28692ce47120bdd9ada8fc23806e897 contains the code with the changes you mentioned, which seems to work now :)

(This method didn't work locally though, might be an issue on my system - sorry for the inconvenience.)

File: tests/test_classification_accs.py [hunk 2]
```diff
@@ -28,3 +27,38 @@ def test_evaluate_multilabel_accuracy(self):
         pred = np.array([0, 1, 1, 1, 1, 1])
         y_pred = {'prediction': pred}
         self.assertAlmostEqual(evaluate_multilabel_accuracy(y_true, y_pred), 0.9090909091, 10)
+
+    def test_evaluate_top_k_accuracy(self):
+        # vacuous case: no error
+        true = np.array([1, 1, 1, 0, 0, 0])
+        y_pred_true = {'prediction': true}
+        self.assertEqual(evaluate_top_k_accuracy(true, y_pred_true, k=1), 1.0)
```
Comment (paxcema) [inline tests/test_classification_accs.py]: It would be good to add another assertion in each case here to check against the actual value that the `sklearn` implementation returns as well

**Review by paxcema — CHANGES_REQUESTED** [review]

File: tests/test_classification_accs.py (hunk 1 above)
Comment (paxcema) [reply tests/test_classification_accs.py]: No worries, glad it's working now! Left one more request and then I think we're ready to merge.

File: tests/test_classification_accs.py (hunk 2 above)
Comment (vijay-jaisankar) [reply tests/test_classification_accs.py]: Thanks for the feedback! Please see https://github.com/mindsdb/mindsdb_evaluator/pull/16/commits/a9b7358935582ef447da5e82a9224902afff1915 which includes these checks too 😄

**Review by paxcema — APPROVED** [review]
Thanks for your contribution @vijay-jaisankar! We'll get this merged and into `mindsdb` proper as soon as possible 🥳
